masschallenge/django-accelerator
# [AC-9413] Update models related to profiles

## [AC-9413](https://masschallenge.atlassian.net/browse/AC-9413)

**Testing**
- Confirm the following requirements are satisfied:
  - A data migration adding Entrepreneur and Subject Matter Expert to the ExpertCategory table exists
  - Change verbose_name on primary_industry field from "Primary Industry"  to "Primary Industry/Experience"
  - Change verbose_name on expert_category field from   "I primarily consider myself a(n)" to "My background is primarily as a(n)"
